src/NUnitFramework/framework/Internal/Execution/ParallelWorkItemDispatcher.cs

```diff
@@ -306,8 +304,12 @@ internal static ExecutionStrategy GetExecutionStrategy(WorkItem work)
                 work.Test is TestFixture && work.Context.ParallelScope.HasFlag(ParallelScope.Fixtures))
                     return ExecutionStrategy.Parallel;
 
-            // If all else fails, run on same thread
-            return ExecutionStrategy.Direct;
+            // There is no scope specified either on the item itself or in the context.
+            // In that case, simple work items are test cases and just run on the same
+            // thread, while composite work items and teardowns are non-parallel.
+            return work is SimpleWorkItem
+                ? ExecutionStrategy.Direct
+                : ExecutionStrategy.NonParallel;
```

Comment:
This is the problem I described in the issue, although not the cause of the user-reported error.

We want to make sure that any work items representing fixtures get a set of isolated queues to run on. I kept the `ExecutionStrategy.Direct` optimization for test cases, since they don't spawn any children.